lib/Basic/PlatformUtility.cpp

```diff
@@ -28,6 +28,11 @@
 #endif
 #include <stdio.h>
 
+#if defined(_WIN32)
+const HANDLE llbuild::basic::sys::FileDescriptorTraits<HANDLE>::InvalidDescriptor =
+    INVALID_HANDLE_VALUE;
```

Comment:
I don't know if it would work but would it work to just define `static const ... = INVALID_HANDLE_VALUE` up there in the trait itself? I would prefer not to split this logic between different files. Is there a particular reason for the approach taken in this patch?

Reply:
It doesn't actually work in the inline constructor (that was what I tried first), hence the split up.  Putting the definition in the header results in the value being defined multiply which also doesn't work.